rs/replicated_state/src/page_map/page_allocator.rs

```diff
@@ -84,6 +89,11 @@ impl PageAllocator {
     /// iterator. Knowing the page count beforehand allows the page allocator
     /// to optimize allocation.
     pub fn allocate(&self, pages: &[(PageIndex, &PageBytes)]) -> Vec<(PageIndex, Page)> {
+        // If the pages that need to be allocated and copied are more than 100 MB,
```

Comment:
100MB?

Reply:
Sorry was looking at an old version.

Reply:
yes, thanks, I noticed that as well. Fixed now.